sass/main.scss

```diff
@@ -940,6 +938,57 @@ footer {
         grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
+
+}
+
+@media screen and (min-width: 1700px) {
+    nav {
+        font-size: 1.5em;
+
+        #login {
+	    padding: 15px 30px;
+	}
+
+	#logo {
+	    width: 120px;
+	    height: 120px;
+	}
+
+	#mlh-badge{
+	    width: 140px;
+	}
+
+    }
+
+    #hero {
+        #desk {
+	    height: 110%;
+	    width: 110%;
+	    top: -50px;
+	}
+
+        #shapes {
+	    height: 1000px;
+	    width: 1000px;
+	    margin-top: -140px;
+	}
+
+        #bh7 {
+            font-size: 6.4em;
+        }
+    
+        #premiere {
+            font-size: 3em;
+        }
+
+        #dates {
+            font-size: 1.57em;
```

Comment:
Why _exactly_ `1.57`? `1.6` should be good too -- let's try to stick to tenths place nums unless absolutely necessary.

Reply:
[resolved within this commit](https://github.com/codeRIT/brickhack.io/pull/1053/commits/cb54bb2673be9e092417dba2a09e1423c7d038aa)